docs/GETTING_STARTED.md

```
@@ -640,6 +646,113 @@ curl -I https://open-inspect-web-{deployment_name}.YOUR-SUBDOMAIN.workers.dev
+
+| Variable               | Drives                                                                                                                   |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `web_app_domain`       | `NEXTAUTH_URL` (Cloudflare web_platform only)                                                                            |
+| `control_plane_domain` | `NEXT_PUBLIC_WS_URL`, `CONTROL_PLANE_URL`, and Modal's `ALLOWED_CONTROL_PLANE_HOSTS` (when `sandbox_provider = "modal"`) |
+
+### 3. Update the GitHub App callback URL
+
+Before you apply, add a new line to the GitHub App's "Identifying and authorizing users" → Callback
+URL field:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)